Resources/Textures/Interface/AdminNotes/LICENCE

```diff
@@ -0,0 +1,4 @@
+https://github.com/tgstation/tgstation/tree/16bb4b376451ab97bcd8e6e014953a0a34f5ab38/icons/ui_icons/notes
+Four files from this folder were sourced from the link above. All credit for those goes to their respective author.
+watchlist.png and message.png were made by Riggle. Same licence applies.
+Licenced under Creative Commons 3.0 BY-SA.
```

Comment:
This should be a .rga attributions file, not a LICENSE file.

Reply:
I hope this is better

Reply:
No, RGA. https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/LobbyScreens/attributions.yml for example.

Reply:
God I love how nothing is standard, I found three different ways to do it already

Reply:
Done, should be good now